sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkConfiguration.java

```diff
@@ -47,6 +47,10 @@
   @JsonProperty
   public NexmarkUtils.SinkType sinkType = NexmarkUtils.SinkType.DEVNULL;
 
+  /** Shall we export the summary to BigQuery. */
```

Comment:
How about "If true, exports the summary to BigQuery." ?

Reply:
If false, the summary is only output to the console.
If true the summary is output to the console and it's content is written to bigquery tables per query x runner x mode.

Reply:
Please rewrite this comment (sounds like a TODO currently). What you wrote above is fine.

Reply:
+1